docs/editor_integration.md

```diff
@@ -233,6 +235,16 @@ If you later want to update _Black_, you should do it like this:
 $ pip install -U black --no-binary regex,typed-ast
 ```
 
+### With ALE
+
+1. Install [`ale`](https://github.com/dense-analysis/ale)
+2. Install `black`
+3. Add this to your vimrc:
+
+   ```vim
+   let g:ale_fixers = {}
+   let g:ale_fixers.python = ['black']
+   ```
```

Comment:
*sigh*, this still fails the lint status check. I would fix it myself, but I can't... @JelleZijlstra can you add an extra line after this end of code fence?

Reply:
done